packages/secret-scan/src/types/index.d.ts

```diff
@@ -7,3 +7,8 @@ export interface SecretResult {
   found: boolean
   regex?: RegExp
 }
+
+export interface ScanObjectResult {
+  secrets: string[][] // string[] -> [key, value]
```

Comment:
I think a better structure would be to use `Record<string, string>`

Reply:
I initially went with this. Then I revisited the #503 requirements and they were arrays. We could use a separator '=' to make it string[].
Which one should I proceed with?

Reply:
I think Record should work fine